src/main/java/com/spotinst/sdkjava/model/api/ocean/kubernetes/ApiK8sVngSpec.java

```diff
@@ -0,0 +1,390 @@
+package com.spotinst.sdkjava.model.api.ocean.kubernetes;
+
+import com.fasterxml.jackson.annotation.JsonFilter;
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.spotinst.sdkjava.client.rest.IPartialUpdateEntity;
+
+import java.util.Date;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@JsonFilter("PartialUpdateEntityFilter")
+public class ApiK8sVngSpec implements IPartialUpdateEntity {
```

Comment:
Please align all variables.
There should be a parameter called "scheduling" as per documentation.

Reply:
There is no "scheduling" in VNG